benchmarks.yml

```diff
@@ -101,6 +101,19 @@ groups:
             decorator-driven, 
             MVC framework for Deno.
 
+      - name: mandarinets_canary
+        version: v1.9.0  
```

Comment:
not canary?

Reply:
otherwise it looks good

Reply:
Canary for mandarine, not for deno. So this one will keep up-to-date the latest changes in master (mandarine)

Reply:
I added the deno version modification though because it looks cool and it's a nice to have in this repo.

Reply:
> otherwise it looks good

great, you can merge. Can you run the pipeline after merging? so we don't have to wait until midnight to see the results :) ?

Reply:
Oh, thanks! yeah sure